caasp-kube-state-metrics-image/caasp-kube-state-metrics-image.kiwi

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v4/kube-state-metrics:%%PKG_VERSION%%-rev<VERSION> caasp/v4/kube-state-metrics:%%PKG_VERSION%%-rev<VERSION>-build<RELEASE> -->
+
+<image schemaversion="6.9" name="caasp-kube-state-metrics-image" xmlns:suse_label_helper="com.suse.label_helper">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>Prometheus running on a SLES15 SP1 container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.1">
+      <containerconfig
+        name="caasp/v4/kube-state-metrics"
+        tag="%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <expose>
+          <port number="8080/tcp"/>
+          <port number="8081/tcp"/>
+        </expose>
+        <entrypoint execute="/usr/bin/kube-state-metrics --port=8080 --telemetry-port=8081"/>
+          <argument name="--port=8080"/>
+          <argument name="--telemetry-port=8081"/>
+        </entrypoint>
+        <subcommand clear="true"/>
+        <labels>
+          <suse_label_helper:add_prefix prefix="com.suse.caasp.v4">
+            <label name="org.opencontainers.image.description" value="Prometheus running on a SLES15 SP1 container guest"/>
```

Comment:
Same as above we should not ship a bunch of images with the same exact description